src/services/notifications.js

```diff
@@ -49,8 +49,15 @@ export const dismiss = id => {
   store.dispatch(actions.setAlerts(alerts.filter(a => a.id !== id)))
 }
 
-export const error = message => {
-  createNotification('error', message)
+export const error = error => {
+  console.log({ error })
```

Comment:
Is this `console.log` _supposed_ to be left in here?

Reply:
Dang it! No its not :smile: good catch @lurch :+1: